index.html

```diff
@@ -47,5 +47,189 @@ <h3>LET'S CONNECT</h3>
           </div>
         </div> 
        </section> 
+
+       <section class="works">
+         <div class="works-flex">
+           <div class="works-flex-item work-image">
+             <img src="snapshoot_portfolio.png" class="img-style" alt="tonic works" />
+           </div>
+           <div class="works-flex-item">
+               <h2 class="intro-works-text">Tonic</h2>
+           </div>
+           <ul class=" works-flex-item titles-container">
+              <li class="list-main">CANOPY</li>
+              <li class="titles-item">Back End Dev</li>
+              <li class="titles-item">2015</li>
+           </ul>
+           <p class="works-flex-item works-description">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+           <div class="works-flex-item">
+             <ul class="lang-container">
+               <li class="lang">html</li>
+               <li class="lang">css</li>
+               <li class="lang">javascript</li>
+             </ul>
+           </div>
+           <div class="works-flex-item">
+             <div class="btn-see">
+               <a href="#" class="btn-see-project">See project</a>
```

Comment:
- [ ] [OPTIONAL] As the previous reviewer mentioned, in order to keep your code clean it could be a good idea to remove one of the `<div>` tags. Remember that it is considered a good practice to avoid using `<div>` and also I think that you are wrapping `one` element in two `<div>`s. If the spacing/layout is the only reason to use two `<div>`s you can use the "parent" `<div>` to apply this as I did.
![image](https://user-images.githubusercontent.com/81522435/143591804-761f2c4c-7f0b-4948-a8b9-82bbe074b3ba.png)
![image](https://user-images.githubusercontent.com/81522435/143591844-0ecf6851-8d14-46e1-8390-a41dc041c452.png)

  **I will mark this as [OPTIONAL] but I strongly suggest removing these tags unless they're completely necessary.**